tests/common/hypothesis-chrome-extension-test.js

```diff
@@ -121,27 +120,26 @@ describe('HypothesisChromeExtension', function() {
       fakeTabState.onChangeHandler = onchange;
     }
     FakeTabState.prototype = fakeTabState;
-
-    HypothesisChromeExtension = proxyquire(
-      '../../src/common/hypothesis-chrome-extension',
-      {
-        './tab-state': FakeTabState,
-        './tab-store': createConstructor(fakeTabStore),
-        './help-page': createConstructor(fakeHelpPage),
-        './browser-action': createConstructor(fakeBrowserAction),
-        './sidebar-injector': createConstructor(fakeSidebarInjector),
-        './errors': fakeErrors,
-        './settings': {
-          serviceUrl: 'https://hypothes.is/',
-        },
-      }
-    );
+    FakeTabState.states = TabState.states;
```

Comment:
Same deal as above, the `states` property was missing from the mock but proxyquire used to call through to the original.